modules/new_profile/jsx/NewProfileIndex.js

```diff
@@ -196,6 +265,41 @@ class NewProfileIndex extends React.Component {
           required = {true}
         />;
     }
+    if (this.state.configData['guidProjects'].includes(
+      this.state.configData.project[this.state.formData.project])
+    ) {
+      GUIDform =
+        <div>
+          <TextboxElement
+            name = "firstName"
+            label = "First Name"
+            onUserInput = {this.setFormData}
+            value = {this.state.formData.firstName}
+            required = {true}
+          />
+          <TextboxElement
+            name = "middleName"
+            label = "Middle Name"
+            onUserInput = {this.setFormData}
+            value = {this.state.formData.middleName}
+            required = {true}
+          />
+          <TextboxElement
+            name = "lastName"
+            label = "Last Name"
+            onUserInput = {this.setFormData}
+            value = {this.state.formData.lastName}
+            required = {true}
+          />
+          <TextboxElement
+            name = "placeOfBirth"
+            label = "Place of Birth (City, Municipality)"
```

Comment:
I'd probably just stick with city here. I don't know if "Municipality" helps clear things up.

Reply:
hmmm I copied this from openscience, have they been using it wrong ? what are the impacts.

we can take this offline for this one